src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1802 @@
+        ),
+        "rr_vaginal_bleeding_cc_stage3": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 3 cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage4": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 4 cervical cancer"
+        ),
+        "prob_referral_biopsy_given_vaginal_bleeding": Parameter(
+            Types.REAL, "probability of being referred for a biopsy if presenting with vaginal bleeding"
+        ),
+        "sensitivity_of_biopsy_for_cervical_cancer": Parameter(
+            Types.REAL, "sensitivity of biopsy for diagnosis of cervical cancer"
+        ),
+        "sensitivity_of_xpert_for_hpv_cin_cc": Parameter(
+            Types.REAL, "sensitivity of xpert for presence of hpv, cin or cervical cancer"
+        ),
+        "sensitivity_of_via_for_cin_cc": Parameter(
+            Types.REAL, "sensitivity of via for cin and cervical cancer bu stage"
+        ),
+        "prob_xpert_screen": Parameter(
+            Types.REAL, "probability of xpert screening"
+        ),
+        "prob_via_screen": Parameter(
+            Types.REAL, "probability of via screening"
+        ),
+        "prob_thermoabl_successful": Parameter(
+            Types.REAL, "probability of thermoablation treatment successful in removing CIN (ce_hpv_cc_status set to none)"
+        ),
+        "prob_cryotherapy_successful": Parameter(
+            Types.REAL, "probability of cryotherapy treatment successful in removing CIN (ce_hpv_cc_status set to none)"
+        ),
+        "transition_testing_year": Parameter(
+            Types.REAL, "year testing recommendation switches from VIA to Xpert"
+        ),
+        "transition_screening_year": Parameter(
+            Types.REAL, "year screening recommendation switches from Cryo to Thermo"
+        ),
+        "min_age_hpv": Parameter(
+            Types.REAL, "minimum age individual can be diagnosed with HPV"
+        ),
+        "screening_min_age_hv_neg": Parameter(
+            Types.REAL, "minimum age individual to be screened if HIV negative"
+        ),
+        "screening_max_age_hv_neg": Parameter(
+            Types.REAL, "maximum age individual to be screened if HIV negative"
+        ),
+        "screening_min_age_hv_pos": Parameter(
+            Types.REAL, "minimum age individual to be screened if HIV positive"
+        ),
+        "screening_max_age_hv_pos": Parameter(
+            Types.REAL, "maximum age individual to be screened if HIV positive"
+        ),
+        "yrs_between_screen_hv_pos": Parameter(
+            Types.REAL, "minimum years between screening if HIV positive"
+        ),
+        "yrs_between_screen_hv_neg": Parameter(
+            Types.REAL, "minimum years between screening if HIV negative"
+        ),
+        "palliative_care_bed_days": Parameter(
+            Types.REAL, "palliative_care_bed_days"
+        ),
+        "stage_1_3_daly_wt": Parameter(
```

Comment:
are these parameters bespoke??

Could they not be got from the HealthBurden module using

```python
sim.modules['HealthBurden'].get_daly_weight(sequlaue_code)```

Reply:
looks like they are!